features/git-town/configuration/configuration_automatic.feature

```diff
@@ -10,68 +10,20 @@ Feature: Automatically running the configuration wizard if Git Town is unconfigu
 
 
   Scenario Outline: All Git Town commands show the configuration prompt if running unconfigured
-    When I run `<COMMAND>` and enter "^C"
+    When I run `<COMMAND>` and enter "main" and ""
     Then I see the initial configuration prompt
+    And the main branch name is now configured as "main"
+    And my perennial branches are configured as none
+    And it may error
```

Comment:
This reads too ambiguous. Does it error or not? Does it only sometimes error? If so, when? Randomly?